plugins/woocommerce/includes/class-wc-emails.php

```diff
@@ -763,6 +763,14 @@ public function low_stock( $product ) {
 			return;
 		}
 
+		// If this is a variation but stock is managed at the parent level, use the parent product for the notification.
+		if ( $product->is_type( 'variation' ) && 'parent' === $product->get_manage_stock() ) {
```

Comment:
Nit: Could use `ProductType::VARIATION`, i.e. `$product->is_type( ProductType::VARIATION )`.

Reply:
Thank you, fixed.